exporters/otlp/otlplog/otlploggrpc/example_test.go

```diff
@@ -31,3 +37,64 @@ func Example() {
 	// From here, the provider can be used by instrumentation to collect
 	// telemetry.
 }
+
+func ExampleWithTLSCredentials() {
```

Comment:
Where is the information that this is an example of a self-signed certificate usage?